### PR TITLE
return cookie on login and check cookie in sse/subscribe

### LINKED ISSUE #144: use cookie for sse authentication
As a developer I would like to use the native eventsource, not a external package for the SSE authentication.

Thus the sse authentication would require a cookie.

### PR DESCRIPTION
**Pull request by cstenglein**
resolves #144

### CODE REVIEW

**Comment by fusion44** [comment]
Hmm, this breaks normal JWT auth. We must find a better solution.

**Comment by cstenglein** [comment]
Added the ability to check for the `Authorization` header as well if no cookie is present, so it doesn't break JWT auth.

**Comment by cstenglein** [comment]
The workaround with the `replace` stems from my limited python knowledge, suggestions appreciated.

File: app/routers/system.py [hunk 1]
```diff
@@ -56,12 +56,16 @@ async def login(i: LoginInput):
             )
             data = parse_key_value_text(result)
             if data["correct"] == "1":
-                return sign_jwt()
+                token = sign_jwt().get("access_token")
+                response.set_cookie("access_token", token)
+                return token
```
Comment (fusion44) [inline app/routers/system.py:61]: Last nitpick: return the whole object from sign_jwt() instead of only the token to not break existing applications.

Comment (cstenglein) [reply app/routers/system.py]: Done.

**Comment by cstenglein** [comment]
It's ok if the code will be refactored or will be changed, just need cookie auth so I can get rid of the dependency mentioned in #144.

**Comment by cstenglein** [comment]
@fusion44 please review again :)

**Pull request merged**